lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
I've mentioned this before, I'm a little concerned that by defaulting these fields we're setting up to mis-categorize somebody. vets.gov is predominantly for veterans but anybody can sign up. I don't know enough to say where in the process a non-veteran will be blocked - guessing they will not have an ICN. But a service_member (who may be eligible for MHV) - will they have an ICN? At some point a service_member switches to becoming a veteran. How does that work?

There's probably some crazy flowchart that maps this all out but have we thought about where the implementation of that business logic will live? I don't think this client is the right place, so more just a general curiosity comment.